README.md

```diff
@@ -121,3 +121,62 @@ For more information on using ember-cli, visit
 
 For more information on using cargo, visit
 [doc.crates.io](http://doc.crates.io/).
+
+## Deploy a read-only, download-API-only mirror
```

Comment:
Could this section perhaps also mention that this is a work-in-progress? Ideally we'd make all deployment a one-click operation, but hopefully coming soon!